addons/event_sale/models/event_ticket.py

```diff
@@ -14,25 +14,41 @@ class EventTemplateTicket(models.Model):
     def _default_product_id(self):
         return self.env.ref('event_sale.product_product_event', raise_if_not_found=False)
 
+    description = fields.Text(compute='_compute_description', readonly=False, store=True)
     # product
     product_id = fields.Many2one(
-        'product.product', string='Product', required=True, domain=[("event_ok", "=", True)],
-        default=_default_product_id)
-    price = fields.Float(string='Price', digits='Product Price')
-    price_reduce = fields.Float(string="Price Reduce", compute="_compute_price_reduce", digits='Product Price')
+        'product.product', string='Product', required=True,
+        domain=[("event_ok", "=", True)], default=_default_product_id)
+    price = fields.Float(
+        string='Price', compute='_compute_price',
+        digits='Product Price', readonly=False, store=True)
+    price_reduce = fields.Float(
+        string="Price Reduce", compute="_compute_price_reduce",
+        digits='Product Price')
 
     @api.depends('product_id')
+    def _compute_price(self):
+        for ticket in self:
+            if ticket.product_id and ticket.product_id.lst_price:
+                ticket.price = ticket.product_id.lst_price or 0
+            elif not ticket.price:
```

Comment:
Same for price and description, I feel like testing "is None" is less confusing.